conans/client/conan_api.py

```diff
@@ -493,7 +519,7 @@ def info_get_graph(self, reference, remote=None, settings=None, options=None, en
 
         profile = profile_from_args(profile_name, settings, options, env, scope, build_folder,
                                     self._client_cache.profiles_path)
-        ret = self._manager.info_get_graph(reference=reference, current_path=current_path,
+        ret = self._manager.info_get_graph(reference=reference,
```

Comment:
How will you get the graph of the current conanfile.txt if current_path to it is not provided?

This could be a big issue.

Reply:
Follow the parameter in develop, it ends only used in loader.py to load_virtual, that assigns `self.conanfile_directory` in the ConanFile constructor. Why is needed in a virtual conanfile to know the path of, what?